### DISCUSSION TITLE
Specify search bar positioning

### DISCUSSION THREAD

**Discussion by mgulyas42** [espanso/espanso#2041] [Q&A] [answered]
Hello guys. I'm new with espanso. I'm using linux with guake terminal. If the terminal is open the search bar will be under the terminal view. Can I change the search bar's position to not show in the middle? or can I just force to show on top even if guake is open?

Thank you!

**Comment by smeech** [comment, marked as answer]
Espanso doesn't enable specifying the position of its Search window, I'm afraid.

Is your guake terminal set to be always on top? Could you give it some transparency?

If it's enough of a problem that you're happy to run a small script in the background, this [script](https://pastebin.com/rh7r7wCV) will wait for the Espanso window to open, and then move it to the top left of your monitor. Make it executable, and adjust it to your liking.

Potentially slightly more efficient [script](https://pastebin.com/iMquAPAS).

**Reply by mgulyas42** [reply]
Yeah guake is running in the background like espanso with a shortcut to display on top. The manual script is far enough for me thank you very much.

**Reply by smeech** [reply]
You're welcome.

Repositioning the window is a fairly frequent request, but the script may help some people until we can get it implemented in Espanso.
